scripts/vendor-boringssl.sh

```diff
@@ -191,7 +191,7 @@ PATTERNS=(
 'crypto/*/*/*.S'
 'crypto/*/*/*/*.c'
 'third_party/fiat/*.h'
-'third_party/fiat/*.c'
+#'third_party/fiat/*.c'
```

Comment:
If this remains commented out and files matching this pattern _do_ exist the next time this is run, will there be an obvious failure?

Reply:
Yeah, the compile will fail due to missing symbols.

Reply:
Cool, thanks